plugins/scanners/fossid/src/main/kotlin/FossIdScanResults.kt

```diff
@@ -156,7 +156,8 @@ private fun mapLicense(
     }.onFailure { spdxException ->
         issues += FossId.createAndLogIssue(
             source = "FossId",
-            message = "Failed to parse license '$license' as an SPDX expression: ${spdxException.collectMessages()}"
+            message = "Failed to parse license '$license' from path '${location.path}' as an SPDX expression: " +
```

Comment:
Hmm, how about slight refactoring `createAndLogIssue()` first so this could populate an `Issue`'s `affectedPath` property?

Reply:
Thanks for the suggestion. I like that.
This also solves the possible issues, that changing the message causes issue resolutions to become invalid.

Reply:
BTW, I would have also been fine with the signature that takes the `affectedPath` before the `severity` (to use the same order as in `Issue` itself), because only a few callers seem to not name the `severity` argument, so these probably can easily be fixed. Leaving it up to you.

Reply:
Issue defines `affectedPath` after `severity`. That was the main reason why I changed the implementation to this order.